index.js

```diff
@@ -1,53 +1,42 @@
 const dogs = []
-document.addEventListener('DOMContentLoaded', () => {
-    fetch('http://localhost:3000/dogs') 
+document.addEventListener("DOMContentLoaded", () => {
+    fetch("http://localhost:3000/dogs")
     .then(response => response.json())
     .then(data => getDogs(data))
    
-    const searchForm = document.getElementById("search-form");
+    const searchForm = document.getElementById("search-form")
```

Comment:
Similar to the quote comment; you had some semicolons, but not everywhere, so I just removed them. Every company and person has a different preference on them, but the best rule is to be consistent.

Reply:
Again, this isn't serious. :)